### PR TITLE
Supply single argument to get kubeconfig.

### PR DESCRIPTION
**Pull request by kwoodson**
Make it easier to get a kubeconfig file by supplying the name of the `$CLUSTER`.

### CODE REVIEW

**Comment by kwoodson** [comment]
This worked to fetch the kubeconfig.
```
./hack/get-admin-kubeconfig.sh kwoodson-e2e
WARN[2020-03-16T11:08:55-04:00]pkg/env/env.go:46 env.NewEnv() running in development mode                  
clusters:
- cluster:
```

This worked to fetch the correct location but was unable to get the nic of the bootstrap as it was already terminated.  The install was successful.
```
./hack/ssh-bootstrap.sh kwoodson-e2e
WARN[2020-03-16T11:12:30-04:00]pkg/env/env.go:46 env.NewEnv() running in development mode                  
WARN[2020-03-16T11:12:37-04:00]pkg/env/env.go:46 env.NewEnv() running in development mode                  
The Resource 'Microsoft.Network/networkInterfaces/aro-bootstrap-nic' under resource group 'aro-tkt82v4d' was not found.
```

File: hack/get-admin-kubeconfig.sh [hunk 1]
```diff
@@ -5,4 +5,6 @@ if [[ "$#" -ne 1 ]]; then
     exit 1
```
Comment (Makdaam) [inline hack/get-admin-kubeconfig.sh]: I'd also change the usage message.
Other than that.
/lgtm

**Comment by jim-minter** [comment]
@olga-mirensky has a different approach in #262 which I'd prefer to take.  It's a little clumsy using full cluster resource IDs but I'd like us to get more used to it because we'll be doing that quite a lot in prod.

**Comment by kwoodson** [comment]
@jim-minter I'm fine with @olga-mirensky's make suggestion.  I would like to know more about how we will be using full cluster resource IDs.  We should be looking at ways to automate this away since the key information is in the subid, resourcegroup, and cluster name.

**Pull request closed without merging**